deploy/packages/bbp-packages.yaml

```diff
@@ -72,13 +72,13 @@ packages:
       - touchdetector@4.4.2
       - touchdetector@5.1.0
       - touchdetector@5.3.2
-      - brayns +opendeck +brion ^ospray@1.7.3
-      - brayns@1.0.1  +brion
+      - brayns +opendeck +brion ^ospray@1.7.3%intel
+      - brayns@1.0.1 +brion ^ospray%intel
       - brayns@immersive +opendeck +brion +optix
-      - brayns-graphs@0.1.0 ^brayns +brion
-      - brayns-topologyviewer@0.1.0 ^brayns@0.8.0 ^ospray@1.7.3
-      - brayns-brainatlas@0.1.0  ^brayns@0.8.0 ^ospray@1.7.3
-      - brayns-membranelessorganelles@0.1.0 ^brayns@0.8.0 ^ospray@1.7.3
+      - brayns-graphs@0.1.0 ^brayns +brion ^ospray%intel
+      - brayns-topologyviewer@0.1.0 ^brayns@0.8.0 ^ospray@1.7.3%intel
```

Comment:
Do we need the version pins on ospray? Do you not run into runtime conflicts mixing things like this?

Reply:
we do unfortunately; ospray 1.7 and 1.8 are not compatible. Sooner or later all plugin will be updated to use brayns@1.0 and we will get rid of 0.8 (and hence ospray 1.7.3)